client/src/components/MainLayout.svelte

```diff
@@ -0,0 +1,45 @@
+<script>
+    import Sidebar from './Sidebar.svelte';
+    import { Router, Route } from 'svelte-routing';
+    import Home from './Home.svelte';
+    import Login from './Login.svelte';
+    import Register from './Register.svelte';
+    import Profile from './Profile.svelte';
+
+    let isLoggedIn = true; // Simulated login state
+
+    function handleLogin() {
+        isLoggedIn = true;
+    }
+
+    function handleLogout() {
+        isLoggedIn = false;
+    }
+</script>
+
+<div class="main-layout">
+    <Sidebar {isLoggedIn} />
+
+    <main class="router-container">
+        <Router>
+            <Route path="/" component={Home} />
```

Comment:
I think this already exists in router.Svelte file.

@Zhangjason308, please confirm